### DISCUSSION TITLE
Connection to SharePoint - inconsistent behaviour

### DISCUSSION THREAD

**Discussion by PavelBrokhman** [pnp/powershell#854] [Q&A]
In our team we are using a lot PnP PowerShell to work with SharePoint. Until recently everyone in our team were able to use Connect-PnPOnline with credentials only. Then one by one different users started to get exception that they must to be consented with PnP Management Shell or use -UseWebLogin -ForceAuthentication. Week ago I also lost this option and I am not sure what I have done that caused it. 

All developers looks like have the same version 3.29. Now there is only 1 developer who can use Connect-PnPOnline in the legacy way. 
We have done few tests. For instant, when I connect remotely to his computer I can connect as well using my credentials. When he connects remotely to my computer we see different error, that says 
`AADSTS53001: Device is not in required device state: domain_joined. Conditional Access policy requires a domain joined device,
and the device is not domain joined.`
How can I determine what are the differences between environments? Why does it work for 1 developer and stop working for others?
I understand that new version allows us to use Graph API, but is there way still using CSOM legacy version?

**Comment by 2023SB2023** [comment]
Did you manage to resolve?
